packages/components/src/modal/aria-helper.ts

```diff
@@ -44,12 +44,22 @@ export function modalize( modalElement?: HTMLDivElement ) {
  * @return Whether the element should not be hidden from screen-readers.
  */
 export function elementShouldBeHidden( element: Element ) {
+	// If the element has the 'hidden' attribute, it should already be hidden from assistive tech
+	if ( element.hasAttribute( 'hidden' ) ) {
```

Comment:
@vk17-starlord can we please combine this check within the return statement as suggested by @afercia

Reply:
ok @up1512001